### PR TITLE
Removed references to CIAO and DAnCE.

### PR DESCRIPTION
**Pull request by ops**
This package contains only ACE and TAO.

### CODE REVIEW

File: ACE/rpmbuild/ace-tao.spec [hunk 1]
```diff
@@ -1,7 +1,6 @@
 # Set the version number here.
 %define ACEVER  6.3.3
 %define TAOVER  2.3.3
-%define CIAOVER 1.3.3
```
Comment (jwillemsen) [inline ACE/rpmbuild/ace-tao.spec]: What about the bin/make_release.py script, that updates also this file and updates CIAOVER, any need for a change there?

Comment (ops) [inline ACE/rpmbuild/ace-tao.spec]: Right. I'll update make_release.py as well.

File: ACE/rpmbuild/ace-tao.spec [hunk 2]
```diff
@@ -75,7 +74,7 @@ Group:        Development/Libraries/C and C++
 URL:          http://www.dre.vanderbilt.edu/~schmidt/ACE.html
 License:      DOC License
 %if 0%{?_with_tao:1}%{?_without_tao:0}
-Source0:      http://download.dre.vanderbilt.edu/previous_versions/ACE+TAO+CIAO-src-%{ACEVER}.tar.gz
+Source0:      http://download.dre.vanderbilt.edu/previous_versions/ACE+TAO-src-%{ACEVER}.tar.gz
```
Comment (jwillemsen) [inline ACE/rpmbuild/ace-tao.spec]: I do agree this is a better source tarball to take, but please check also the debian directory where ACE+TAO+CIAO as tarball is used, also check the make_release.py script for the change of the source. Else we need two different gz files on OBS which also wastes space

Comment (ops) [inline ACE/rpmbuild/ace-tao.spec]: OK, I'll take a look. Do we prefer .bz2 or .gz tarball?

Comment (jwillemsen) [inline ACE/rpmbuild/ace-tao.spec]: Not sure if some packaging only works with one version, maybe just use .gz

**Pull request merged**